examples/fl-tabular/centralized.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import LabelEncoder, StandardScaler
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader, TensorDataset
+
+url = "https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data"
+column_names = [
+    "age",
+    "workclass",
+    "fnlwgt",
+    "education",
+    "education_num",
+    "marital_status",
+    "occupation",
+    "relationship",
+    "race",
+    "sex",
+    "capital_gain",
+    "capital_loss",
+    "hours_per_week",
+    "native_country",
+    "income",
+]
+
+data = pd.read_csv(
+    url, header=None, names=column_names, na_values=" ?", sep=",\s", engine="python"
+)
```

Comment:
why not use flower datasets?

Reply:
I did that intentionally to demonstrate how Flower datasets can be utilized compared to not using them.

Reply:
Sorry, but it does not make full sense to me. It's a point that can confuse users. Also, while theoretically it shouldn't, it brings the possibility of data discrepancy that is unintentional (since it comes from a different source). + Double data download.

Reply:
@adam-narozniak @jafermarq in that case, I think it is better to remove the `centralized.py`.  My idea was to show how centralized code (without the FL framework and dataset) can be transitioned to FL code. But it is getting confusing and I guess better to just remove it. wdyt?

Reply:
how about using HF Datasets to get the dataset. (this is what FDS would use under the hood). This will likely allow to move some of the common functionality (e.g. doing the one-hot stuff, model definition, train/test functions) to a separate Python file. (as you did a while back for the [`examples/fl-dp-sa`](https://github.com/adap/flower/tree/main/examples/fl-dp-sa/fl_dp_sa)) ?

Reply:
@jafermarq I restructured it and removed `centralized.py`. Please have a look.

Reply:
Abstracting from this example, maybe the partitioners parameter could also be None (in the future releases of FDS) to make it easier to use the dataset for the centralized ML (then load_split could be used)?

We can't show all the examples from the different starting points for people who decide to change from centralized to federate. But if there's starting from zero we can show to easily create both.